exasol_integration_test_docker_environment/lib/test_environment/spawn_test_database.py

```diff
@@ -118,6 +119,15 @@ def _create_database_container(self, db_ip_address: str, db_private_network: str
                     ports=ports,
                     runtime=self.docker_runtime
                 )
+
+            if self.db_os_access == DbOsAccess.SSH:
+                files = SshFiles()
+                sshkey = SshKey.from_folder(files.folder)
```

Comment:
Yes, it was itching me as well. 
I will change maker method of class `SshKey` to take an argument of `SshFiles` instead of a bare folder (datatype str or Path).

Originally I created SshFiles to encapsulate the authorized_keys file and its folder for mounting as well. 
But now it turned out, that we can simply copy a string content to the Docker Container and don't need to replicate the file and folder in the host's OS additionally, which also gives the benefit of not needing to encapsulate the folder for potential multiple concurrent instances of containers and environments.